### DISCUSSION TITLE
12:00 am for time picker issue

### DISCUSSION THREAD

**Discussion by francisdwine** [primefaces/community#1713] [PrimeReact] [answered]
when I choose 12:00 AM. a value of 12:00:00 is returned instead of 00:00:00. it also applies until 12:59 AM. is this a bug for time picker

**Comment by melloware** [comment, marked as answer]
I feel like this has been fixed are you using PR 10.6.3?  If not please create a StackBlitz reproducer showing the issue.

**Comment by francisdwine** [comment]
I'll do it later but. To replicate the issue manually type 12:00 am (don't
use the arrows since they're fine)on the calendar time picker then observe
as it changes to 12:00pm

**Reply by melloware** [reply]
Ahh if its typing only you can see there are a bunch of open calendar issues around typing dates in: https://github.com/primefaces/primereact/issues?q=is%3Aissue+is%3Aopen+calendar

**Comment by francisdwine** [comment]
But the issue tho is it returns a date object with 12:00:00 instead of
00:00:00 it happens from 12:00 am to 12:59 am only

**Reply by melloware** [reply]
OK create a StackBlitz and I can take a look.

**Comment by francisdwine** [comment]
Oh you're right I tested it on the official prime react website it seems
fine. I think I need to update my module or something

**Comment by francisdwine** [comment]
Thanks a lot.
